makefile

```diff
@@ -2,7 +2,7 @@ SHELL := /bin/bash
 
 .PHONY: clean test testprofile testcover docs
 
-flist = 0 1 2 3 4 5 6 7 8 9 12 13 14 15 23 24 25 26
+flist = 3 4 5 6 S02 S03 S04 S05 S06 S07 S08 S09 S12 S13 S14 S15
```

Comment:
This is the main change @Farnazmdi @jclagarde , the figures are now named with `S` for supplementary, and they have been moved to the end of the manuscript. You can now just make figures & and onwards without having to start from figure 27. @jclagarde Note that this changes some minor things in `genFigures.py`. I named the respective cartoons from `figure2.svg` to `figureS02.svg`.